baseplate/clients/redis.py

```diff
@@ -271,9 +308,7 @@ def execute(self, **kwargs: Any) -> Any:
             start_time = perf_counter()
             labels = {
                 f"{PROM_LABELS_PREFIX}_command": "pipeline",
-                f"{PROM_LABELS_PREFIX}_client_name": self.connection_pool.connection_kwargs.get(
```

Comment:
Worth noting that this is backwards incompatible, and supported by the connection pool (it takes arbitrary kw args) but I don't think anyone was doing that so I'm fine with this